src/main/java/dk/cngroup/lentils/service/StatusService.java

```diff
@@ -66,6 +80,12 @@ public Boolean isStatusInDbByCypherAndTeam(final Cypher cypher, final Team team)
         return statusRepository.existsStatusByCypherAndTeam(cypher, team);
     }
 
+    private boolean isStatusInDbByCypherAndTeamAndCypherStatus(final Cypher cypher,
```

Comment:
Muzes prosim prejmenovat metodu na `isStatusPresent` nebo neco podobneho? Odstranit z nazvy metody jmena parametru, protoze to je diplicita v kodu a tady uz to vytvari az prilis velky nazev metody.

Reply:
Nooo, když už to komentuješ, co takhle tu privátní metodu smazat úplně a inlajnovat to volání?
Nikde jinde se tahle metoda nepoužívá co vidím.